docs/howto/singletons.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Making a Singleton Contenttype
+---
+Making a Singleton Contenttype
+===============================
+
+In a nutshell: a Singleton is a contenttype that contains only 1 post.
+You can use singletons for all kinds of things, from complex homepages, singlepage applications to a general settings page for your website.
+
+When you set a contenttype to `singleton: true`, update some aspects of the layout for that posttype to make your experience more fluent.
+
+Creating the settings contenttype
+---------------------------------
+In this example we'll run you trough creating a settings contenttype, here you can allow your site manager to manage things like social media url's, non-page specific content and anything you feel a user should be able to manage.
+
+```yaml
+your_settings:
+    name: Your settings
+    singular_name: Your setting
+    fields:
+        title:
+            type: text
+            class: large
+            group: "Block"
+        slug:
+            type: slug
+            uses: [ title ]
+        header_usps:
+            type: repeater
+            group: header
+            fields:
+                usp_title:
+                    type: text
+        footer_text:
+            type: html
+            group: footer
+        footer_copyright:
+            type: text
+        company_address:
+            type: geolocation
+            group: common
+        
+    viewless: true
+    singleton: true
+    default_status: published
+    searchable: false
```

Comment:
`searchable: false` should not be needed, if we already have `viewless: true`.

Reply:
Ah, then we should probably update the resource contenttype as well, it contains the same example